modules/rubiconBidAdapter.js

```diff
@@ -211,6 +213,17 @@ export const converter = ortbConverter({
     }
 
     setBidFloors(bidRequest, imp);
+    // ensure unique imp IDs for twin adunits
+    var adUnitCodeCount = {};
+    var adUnitCode = imp.id;
+    if (adUnitCodeCount[adUnitCode]) {
+      adUnitCodeCount[adUnitCode] += 1;
+      imp.id = adUnitCode + adUnitCodeCount[adUnitCode];
+    } else {
+      adUnitCodeCount[adUnitCode] = 1;
+    }
+    // update adunit map so we can translate back to original adunit code
+    impIdMap[imp.id] = adUnitCode;
```

Comment:
not sure we need both adUnitCodeCount and impIdMap

would this just work something like

```js
const adUnitCode = imp.id;
if (impIdMap[adUnitCode]) {
      imp.id = `${adUnitCode}-${impIdMap[adUnitCode]}`; // I like adding a - to keep them separate
      impIdMap[adUnitCode] += 1;
} else {
     impIdMap[adUnitCode] = 1;
}
```

Then just reset like you do after `toORTB `
```
impIdMap = {};
```